src/style.css

```diff
@@ -0,0 +1,110 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.margint_indent {
+  margin-top: 20px;
+}
+
+.search_form {
+  display: block;
+  position: relative;
+  height: 70px;
+  margin-bottom: 20px;
+}
+
+.search_svg {
+  position: absolute;
+  height: 19px;
+  width: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.input_big_search_bar {
+  font-family: Avenir-book, Arial, Helvetica, sans-serif;
```

Comment:
move it into html selector and inherit here

Reply:
+

Reply:
I don't know why font family doesn't inherits from html